webservices/partition/sched_b.py

```diff
@@ -79,3 +79,10 @@ def index_factory(cls, child):
     def update_child(cls, child):
         cmd = 'alter table {0} alter column recipient_st set statistics 1000'.format(child.name)
         db.engine.execute(cmd)
+
+    @classmethod
+    def create_trigger(cls):
+        db.engine.execute('DROP TRIGGER IF EXISTS insert_sched_a_trigger ON ofec_sched_b_master')
+        db.engine.execute('''
+            CREATE trigger insert_sched_a_trigger BEFORE INSERT ON ofec_sched_b_master FOR EACH ROW EXECUTE PROCEDURE insert_sched_master('ofec_sched_b_');
+            ''')
```

Comment:
This is what I get for reviewing before I've had ☕️ - I see this is correct now and the trigger names are as they should be, and the function name to execute when they're fired matches as well.